core/blockchain.go

```diff
@@ -2646,3 +2653,7 @@ func (bc *BlockChain) SetTrieFlushInterval(interval time.Duration) {
 func (bc *BlockChain) GetTrieFlushInterval() time.Duration {
 	return time.Duration(bc.flushInterval.Load())
 }
+
+func (bc *BlockChain) NoTries() bool {
```

Comment:
There is only two place that call `NoTries`, however there are six place that call `NoTries`, I believe something is missing.

Reply:
fixed the issue according to the latest bsc code